src/os/ui/query/areafilteradd.js

```diff
@@ -591,8 +591,9 @@ os.ui.query.AreaFilterAddCtrl.prototype.onAreasSelected_ = function(features) {
  * @param {string} label
  * @param {boolean=} opt_hideArea
  * @param {boolean=} opt_hideFilter
+ * @param {boolean=} opt_modal
  */
-os.ui.query.AreaFilterAddCtrl.launch = function(label, opt_hideArea, opt_hideFilter) {
+os.ui.query.AreaFilterAddCtrl.launch = function(label, opt_hideArea, opt_hideFilter, opt_modal) {
```

Comment:
This isnt needed anymore right?

Reply:
it is still needed. it's a window, not a menu.

Reply:
oh since the window you are opening is already a modal, you need this to optionally be a modal so it can open on top of it.

Reply:
correct.